src/smart-components/Message/components/TextFragment/index.tsx

```diff
@@ -21,43 +22,37 @@ export default function TextFragment({
   const message = messageStore?.message as UserMessage;
   const isByMe = messageStore?.isByMe;
   const { updatedAt, createdAt } = message;
+
   return (
     <>
-      {
-        tokens?.map((token, idx) => {
-          const key = keyGenerator(createdAt, updatedAt, idx);
-          switch (token.type) {
-            case TOKEN_TYPES.mention:
-              return (
-                <span className="sendbird-word" key={key}>
-                  <MentionLabel
-                    mentionTemplate={USER_MENTION_PREFIX}
-                    mentionedUserId={token.userId}
-                    mentionedUserNickname={token.value}
-                    isByMe={isByMe}
-                  />
-                </span>
-              );
-            case TOKEN_TYPES.url:
-              return (
-                <span className='sendbird-word' key={key}>
-                  <LinkLabel
-                    className="sendbird-word__url"
-                    src={token.value}
-                    type={LabelTypography.BODY_1}
-                  >
-                    {token.value}
-                  </LinkLabel>
-                </span>
-              );
-            case TOKEN_TYPES.string:
-            default:
-              return (
-                <React.Fragment key={key}>{token.value}</React.Fragment>
```

Comment:
>Also, a sent message which contains white spaces at the beginning of it is shown like it's trimmed.

^^ was due to we used `<React.Fragement />` here. So I replaced it with `<span className='sendbird-word'` like other cases.

Reply:
That was intentional, we wanted the text outside those `span`

Reply:
@sravan-s Oh then do you want me to take this out again?

Reply:
Maybe we can find another work around not to trim the leading white spaces. I wrapped the text \w the span tag because of it.

Reply:
Found a workaround https://github.com/sendbird/sendbird-uikit-react/pull/487/commits/f4f7c3f6b15afd1d7afdc0ea0f65ec4fde9f9f17#diff-69d5dc1d02cceaf88dca2f7556572756c09e171261c4becc6c98d982c1049739R52 not sure it's a best option